services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_assembly.py

```diff
@@ -0,0 +1,135 @@
+from copy import deepcopy
+from typing import Any, Dict, Optional
+
+import yaml
+from aiohttp.web import Application
+
+from .config import DynamicSidecarSettings, get_settings
+from ...models.domains.dynamic_sidecar import PathsMappingModel, ComposeSpecModel
+
+CONTAINER_NAME = "container"
+BASE_SERVICE_SPEC: Dict[str, Any] = {
+    "version": "3.7",
+    "services": {CONTAINER_NAME: {}},
+}
+
+
+def _inject_traefik_configuration(
+    service_spec: Dict[str, Any],
+    target_container: str,
+    dynamic_sidecar_network_name: str,
+    simcore_traefik_zone: str,
+    service_port: int,
+) -> None:
+    """Injects configuration to allow the service to be accessible on the uuid.services.SERVICE_DNS"""
+
+    # add external network to existing networks defined in the container
+    service_spec["networks"] = {
+        dynamic_sidecar_network_name: {
+            "external": {"name": dynamic_sidecar_network_name},
+            "driver": "overlay",
```

Comment:
could you also make it attachable? then we can attach a container at runtime if necessary

Reply:
yes